lib/core/presentation/pages/event/event_detail_page/host_event_detail_page/widgets/host_rewards_section.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:app/core/domain/event/entities/event.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:dotted_border/dotted_border.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class HostRewardsSection extends StatelessWidget {
```

Comment:
This file unused for now, waiting Rishi confirm the UI, then enhance and put into host event detail, this just UI only